browser/ui/commands/accelerator_service.h

```diff
@@ -41,6 +41,7 @@ class AcceleratorService : public mojom::CommandsService, public KeyedService {
   ~AcceleratorService() override;
```

Comment:
Sorry, I can't leave comment for invisibile part so leave it here.

At the line 28, we have Accelerators alias, which is same with alias in the component. Can we replace the line with `using Accelerators = AcceleratorPrefManager::Accelerators;` for consistency? Or making a file like "commmon/commands_types.h" and putting the alias there could be an option.

Reply:
Good call - I'm doing `using Accelerators = AcceleratorPrefManager::Accelerators` because `using` doesn't seem to like `COMPONENT_EXPORT`